internal/oc/release.go

```diff
@@ -62,7 +62,7 @@ func NewRelease(executer executer.Executer, config Config) Release {
 
 const (
 	templateGetImage        = "oc adm release info --image-for=%s --insecure=%t %s"
-	templateGetVersion      = "oc adm release info -o template --template '{{.metadata.version}}' --insecure=%t %s"
+	templateGetVersion      = "oc adm release info -o template --template '{{.metadata.version}}' --filter-by-os=linux/amd64 --insecure=%t %s"
```

Comment:
Is this not something we use with arm or other architectures?
I'm not sure exactly what `filter-by-os` does (I don't see it in my `oc` help output).

Reply:
I think this is not even needed anymore. I'm just removing it and rerunning the E2E multiarch ZTP test